pkg/pipeline/transform/kubernetes/enrich.go

```diff
@@ -142,3 +163,36 @@ func objectIsApp(namespace, name string, rule *api.K8sInfraRule) bool {
 	}
 	return true
 }
+
+// shouldInclude determines if an item should be included based on inclusion/exclusion maps.
+func shouldInclude(key string, inclusions, exclusions map[string]struct{}) bool {
+	if _, excluded := exclusions[key]; excluded {
+		return false
+	}
+	if len(inclusions) > 0 {
+		_, included := inclusions[key]
+		return included
+	}
+	return true
+}
+
+// truncateWithSuffix truncates s to max runes, including the suffix.
+func truncateWithSuffix(s string, max *int, suffix string) string {
+	if max == nil {
+		return s
+	}
+	m := *max
```

Comment:
Fixing linter complaining:

```suggestion
func truncateWithSuffix(s string, maxRunes *int, suffix string) string {
	if maxRunes == nil {
		return s
	}
	m := *maxRunes
```